Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM yarnpkg/node-yarn
+
+
+ENV ANGULAR_DIR /opt/rf-docs
+WORKDIR /opt/rf-docs
+
+COPY yarn.lock package.json ${ANGULAR_DIR}/
+
+RUN yarn install --pure-lockfile
```

Comment:
After successfully installing the Node dependencies, I think that we need to copy in the source code as well:

```dockerfile
COPY . /opt/rf-docs
```